src/style/mod.rs

```diff
@@ -453,81 +431,81 @@ impl FlexboxLayout {
     /// If the `direction` is row-oriented, the min width. Otherwise the min height
     pub(crate) fn min_main_size(&self, direction: FlexDirection) -> Dimension {
         if direction.is_row() {
-            self.min_size.width
+            self.min_size.clone().width
         } else {
-            self.min_size.height
+            self.min_size.clone().height
```

Comment:
Clones here should be on just the dimension being returned. Or is possible, this should return a reference.

Reply:
They now return a reference.